common/src/main/java/com/google/udmi/util/CommandLineOption.java

```diff
@@ -33,4 +33,9 @@
    * Set the argument type description.
    */
   String arg_type() default NO_VALUE;
```

Comment:
I would deprecate/remove arg_type() here since there's no current use/need for it.

Reply:
This is done